Solutions/Digital Shadows/Playbooks/DigitalShadowsPlaybook-UpdateIncidentStatus.json

```diff
@@ -0,0 +1,409 @@
+                                        },
+                                        "mitigation":  {
+                                            "items":  {
+                                                "type":  "string"
+                                            },
+                                            "type":  "array"
+                                        },
+                                        "risk_alert":  {
+                                            "items":  {
+                                                "type":  "string"
+                                            },
+                                            "type":  "array"
+                                        },
+                                        "risk_incident":  {
+                                            "items":  {
+                                                "type":  "string"
+                                            },
+                                            "type":  "array"
+                                        },
+                                        "status":  {
+                                            "items":  {
+                                                "type":  "string"
+                                            },
+                                            "type":  "array"
+                                        },
+                                        "triage_id":  {
+                                            "items":  {
+                                                "type":  "string"
+                                            },
+                                            "type":  "array"
+                                        }
+                                    },
+                                    "type":  "object"
+                                }
+                            }
+                        }
+                    },
+                    "outputs":  {
+                    }
+                },
+                "parameters":  {
+                    "$connections":  {
+                        "value":  {
+                            "azuresentinel":  {
+                                "connectionId":  "[resourceId('Microsoft.Web/connections', variables('azuresentinelConnectionName'))]",
+                                "connectionName":  "[variables('azuresentinelConnectionName')]",
+                                "id":  "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/azuresentinel')]",
+                                "connectionProperties":  {
+                                    "authentication":  {
+                                        "type":  "ManagedServiceIdentity"
+                                    }
+                                }
+                            },
+                            "azuresentinel_1":  {
```

Comment:
It doesn't look right that we have two `azuresentinel` connections in here - this one seems to have a hardcoded subscription ID and resource group.

There are also a number of steps in the playbook that are using this connection. If this is an error then those will need updating as well.

Reply:
@v-sabiraj @sreedharande should I remove one of them? This was generated by the tool which is in the repo for generating the ARM for playbooks.

Reply:
@shikhin-metron: There are couple of issues in the Playbook ARM Template
1. There were two azuresentinel connections, remove one and update the playbook with correct azuresentinel connection
2. Update trigger to Incident trigger, unless there is a strong use case for the alert trigger
3. Did you used ARM Template Generator Tool to create ARM Template for your Playbook?? Metadata is different than what we are expecting

Reply:
Hi, for above issues,
1. Not really sure how to do that if you are asking to do it on portal and then generate playbook again, can you please send a documentation if possible?
2. We are using alert trigger because we update the status or severity when it changes on Digital Shadows end, so changes are group among alerts in an incident, which helps us change status of incident using alert info.
3. We did use the ARM template generator tool for playbook. What kind of metadata are we expecting?

Reply:
@sreedharande about metadata, we had modified it according to the product details. Please suggest us if we need to change anything with respect to the Metadata.